splashsurf/src/io.rs

```diff
@@ -167,3 +160,22 @@ pub fn to_binary_f32<R: Real, P: AsRef<Path>>(file: P, values: &[R]) -> Result<(
 
     Ok(())
 }
+
+// #[repr(transparent)]
+struct Vec3r(Vector3<f32>);
+
+impl ply::PropertyAccess for Vec3r {
+    fn new() -> Self {
+        Self {
+            0: { Vector3::new(0.0, 0.0, 0.0) },
+        }
```

Comment:
Maybe a better default is to use NAN here? So that if properties are missing below, it is not as silent as it would be with zeros? Btw. you can write
```rust
Self(Vector3::repeat(std::f32::NAN))
```